build/docker-exec

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]] ; then
+  echo 'build/docker-exec - run a rake command on a specified ruby version'
+  echo 'Usage:'
+  echo '  build/docker-exec 2.2.5 spec'
+  exit 1
+fi
+
+ruby_version="$1"
+rake_task="$2"
+
+cp -r /app ..
```

Comment:
Is the reason you copy things into your current directory because of how `bundler` creates the `.bundle` directory? If so I wonder if there's a way to turn that off.